tests/e2e/release/cht-config/generate-short-codes.wdio-spec.js

```diff
@@ -0,0 +1,84 @@
+const utils = require('../../../utils');
+const sUtils = require('../../sentinel/utils');
+const commonElements = require('../../../page-objects/common/common.wdio.page');
+const loginPage = require('../../../page-objects/login/login.wdio.page');
+const reportsPage = require('../../../page-objects/reports/reportsPage.wdio');
+const personFactory = require('../../../factories/cht/contacts/person');
+const place = require('../../../factories/cht/contacts/place');
+const places = place.generateHierarchy();
+const clinic = places.find((place) => place.type === 'clinic');
+
+const contact = personFactory.build(
+  {
+    parent: {
+      _id: clinic._id,
+      parent: clinic.parent
+    },
+    phone: '+254712345670'
+  });
+
+const docs = [...places, contact];
+
+describe('generating short codes', () => {
+  const submit =  (body) => {
+    return  utils.request({
+      method: 'POST',
+      path: '/api/v2/records',
+      headers: {
+        'Content-type': 'application/json'
+      },
+      body: body
+    });
+  };
+
+  const form = {
+    'CASEID': {
+      'meta': { 'code': 'CASEID', 'icon': 'icon-healthcare', 'translation_key': 'Case Id Form' },
+      'fields': {},
+      'use_sentinel': true
```

Comment:
Just fyi, `use_sentinel` is never actually used in code and is a relic from past times, it's not been used since 2014? https://github.com/medic/cht-core/issues/500